ToMigrate/Raven.Tests.Issues/RavenDB_576.cs

```diff
@@ -1,31 +0,0 @@
-using Raven.Tests.Bundles.Replication;
-using Raven.Tests.Common;
-
-namespace Raven.Tests.Issues
-{
-    using Xunit;
-
-    public class RavenDB_576 : ReplicationBase
```

Comment:
No, it isn't going to be in 4.0 at all.
Compression is already at the blittable level.